Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1252,6 +1252,7 @@ private static void QueryScene(IMixedRealityPointer pointer, IMixedRealityRaycas
                             }
                             break;
                         case SceneQueryType.SphereOverlap:
+                        case SceneQueryType.ConeOverlap:
```

Comment:
Why is the cone overlap calculation the same as the sphere overlap calculation, in this case? I would expect the cone overlap to be different.